src/map/pc.c

```diff
@@ -2811,16 +2839,34 @@ int pc_bonus2(struct map_session_data *sd,int type,int type2,int val)
 				sd->arrow_addsize[type2]+=val;
 			break;
 		case SP_SUBELE:
-			if(type2 >= ELE_MAX) {
+			if(type2 >= ELE_MAX && type2 != ELE_ALL) {
 				ShowError("pc_bonus2: SP_SUBELE: Invalid element %d\n", type2);
```

Comment:
Curious, this sanity check is only done for this, while it's not done for most others. If it's needed, we should probably add it to the other cases (otherwise remove it from here as well)